docs/src/modules/components/MaterialFreeTemplatesCollection.js

```diff
@@ -115,7 +116,8 @@ export default function Templates() {
                   component="img"
                   // The image source is generated from `pnpm template:screenshot material-ui`, do not modify the image manually.
                   image={`/static/screenshots${layout.href.replace(/\/$/, '')}.jpg`}
-                  title={layout.title}
+                  alt={layout.title}
+                  fetchPriority={index === 0 ? 'high' : undefined}
```

Comment:
I wasn't sure  that we wanted to use loading="lazy" so went for this instead:

<img width="537" alt="SCR-20241119-baix" src="https://github.com/user-attachments/assets/082d5262-5071-494e-a61e-62150e25621c">

Reply:
Oh, annoying https://github.com/vercel/next.js/issues/65161 only really works with React 19.